src/rmp/include/rmp/util.h

```diff
@@ -0,0 +1,23 @@
+#ifndef UTIL_H_
+#define UTIL_H_
+
+#include <iostream>
+#include <fstream>
+#include <vector>
+#include <algorithm>
+#include <unordered_map>
```

Comment:
Nothing in this header looks to require any of these.  You do need string and pair though.